packages/graphql-server/src/plugins/useRedwoodLogger.ts

```diff
@@ -194,6 +194,13 @@ export const useRedwoodLogger = (
   const includeQuery = loggerConfig?.options?.query
 
   return {
+    onPluginInit(context) {
+      context.registerContextErrorHandler(({ error }) => {
+        if (error) {
+          childLogger.error(`Error building context. ${error}`)
```

Comment:
Unfortunately, the error here is `unknown` and doesn't seem to have a message on it.